18.05.2015 12:27:39 - INFO: Alarm!
18.05.2015 12:27:39 - DEBUG: Loading plugin template
18.05.2015 12:27:39 - DEBUG: Throw Template Plugin
18.05.2015 12:27:39 - DEBUG: try 5/0
18.05.2015 12:27:39 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 7, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 12:27:40 - INFO: Alarm!
18.05.2015 12:27:40 - DEBUG: Loading plugin template
18.05.2015 12:27:40 - DEBUG: Throw Template Plugin
18.05.2015 12:27:40 - DEBUG: try 5/0
18.05.2015 12:27:40 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 7, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 12:27:41 - INFO: Alarm!
18.05.2015 12:27:41 - DEBUG: Loading plugin template
18.05.2015 12:27:41 - DEBUG: Throw Template Plugin
18.05.2015 12:27:41 - DEBUG: try 5/0
18.05.2015 12:27:41 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 7, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 12:27:42 - INFO: Alarm!
18.05.2015 12:27:42 - DEBUG: Loading plugin template
18.05.2015 12:27:42 - DEBUG: Throw Template Plugin
18.05.2015 12:27:42 - DEBUG: try 5/0
18.05.2015 12:27:42 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 7, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 12:27:43 - INFO: Alarm!
18.05.2015 12:27:43 - DEBUG: Loading plugin template
18.05.2015 12:27:43 - DEBUG: Throw Template Plugin
18.05.2015 12:27:43 - DEBUG: try 5/0
18.05.2015 12:27:43 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 7, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 12:27:44 - INFO: Alarm!
18.05.2015 12:27:44 - DEBUG: Loading plugin template
18.05.2015 12:27:44 - DEBUG: Throw Template Plugin
18.05.2015 12:27:44 - DEBUG: try 5/0
18.05.2015 12:27:44 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 7, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 12:27:45 - INFO: Alarm!
18.05.2015 12:27:45 - DEBUG: Loading plugin template
18.05.2015 12:27:45 - DEBUG: Throw Template Plugin
18.05.2015 12:27:45 - DEBUG: try 5/0
18.05.2015 12:27:45 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 7, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 12:27:46 - INFO: Alarm!
18.05.2015 12:27:46 - DEBUG: Loading plugin template
18.05.2015 12:27:46 - DEBUG: Throw Template Plugin
18.05.2015 12:27:46 - DEBUG: try 5/0
18.05.2015 12:27:46 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 7, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 12:27:47 - INFO: Alarm!
18.05.2015 12:27:47 - DEBUG: Loading plugin template
18.05.2015 12:27:47 - DEBUG: Throw Template Plugin
18.05.2015 12:27:47 - DEBUG: try 5/0
18.05.2015 12:27:47 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 7, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 12:27:48 - INFO: Alarm!
18.05.2015 12:27:48 - DEBUG: Loading plugin template
18.05.2015 12:27:48 - DEBUG: Throw Template Plugin
18.05.2015 12:27:48 - DEBUG: try 5/0
18.05.2015 12:27:48 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 7, in run
    test = 5/0
ZeroDivisionError: division by zero
18.05.2015 12:27:49 - INFO: Alarm!
18.05.2015 12:27:49 - DEBUG: Loading plugin template
18.05.2015 12:27:49 - DEBUG: Throw Template Plugin
18.05.2015 12:27:49 - DEBUG: try 5/0
18.05.2015 12:27:49 - ERROR: Error in Template Plugin
Traceback (most recent call last):
  File "./plugins\template\__init__.py", line 7, in run
    test = 5/0
ZeroDivisionError: division by zero
